### PR TITLE
Added Button component

### PR DESCRIPTION
**Pull request by borkarsaish65**
image

### CODE REVIEW

File: component/button/button.css [hunk 1]
```diff
@@ -0,0 +1,107 @@
+.btn {
+    display: inline-block;
+    font-size: 1rem;
+    padding:0.3125rem 1rem;
```
Comment (mohammed-sas) [inline component/button/button.css]: for values less than 10px use absolute units

**Review by mohammed-sas — APPROVED** [review]

File: component/button/button.css [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+    border-radius: 100%;
+    border: 1px solid transparent;
+    background-color: var(--primary-color);
+    color:white;
+    height: 1em;
+    width: 1.2em;
+    box-shadow: 3px 4px 10px -2px rgba(0,0,0,0.75);
+    -webkit-box-shadow: 3px 4px 10px -2px rgba(0,0,0,0.75);
+    -moz-box-shadow: 3px 4px 10px -2px rgba(0,0,0,0.75);
+}
```
Comment (AnkurChunekar) [inline component/button/button.css:44]: You can use variables for box shadow property above (DRY principle), it'll help you to be consistent also.

File: component/button/button.css [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+    color:var(--primary-color);
+    background-color: var(--white);
+    border-color: #0d6efd;
+}
+/* .btn-floating.btn-large
+{
+    padding: 1.25rem;
+
+} */
+
```
Comment (AnkurChunekar) [inline component/button/button.css]: remove unnecessary comments

File: component/button/button.css [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+    font-size: 50%;
+} */
+
+/* .btn-floating.btn-large i 
+{
+  display: flex;
+  justify-content: center;
+  align-items: center;
+} */
+
```
Comment (AnkurChunekar) [inline component/button/button.css:63]: here also remove unnecessary commented code

File: component/button/button.css [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+    background: white;
+    text-decoration: underline;
+    cursor:pointer;
+}
+
+.btn-FAB {
+    padding:1rem;
+    border-radius: 100%;
+    background : #82ff9e; 
+} */
```
Comment (AnkurChunekar) [inline component/button/button.css]: Add an empty line at last to avoid this error.

File: component/button/button.html [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+
+   <!--
+    <button type='button' class='btn btn-default'>Click</button>
+
+    <button class='btn btn-link'>Click</button>
+
+
+    <button class='btn btn-FAB'>Click</button> -->
+     <!-- <button class='btn btn-default'>Click</button>
+
```
Comment (AnkurChunekar) [inline component/button/button.html]: Remove unnecessary comments from HTML files also.

File: component/button/button.html [hunk 7]
```diff
@@ -0,0 +1,103 @@
...
+   <button type='button' class='btn btn-primary'>
+    <i class="far fa-bell"></i> 
+   Alert Me
+   </button>
+
+
+
+
+
+   <button type='button' class='btn-floating'>
```
Comment (AnkurChunekar) [inline component/button/button.html]: try to be consistent with line spacing, you can use code formatter like prettier.

File: main.css [hunk 8]
```diff
@@ -0,0 +1,5 @@
+:root {
+    --primary-color: #2563ebff;
+    --secondary-color:#095256ff;
+    --white: #ffffff;
+  }
```
Comment (AnkurChunekar) [inline main.css]: Again you add an empty last line to all of your files.

**Review by AnkurChunekar — CHANGES_REQUESTED** [review]

**Review by AnkurChunekar — APPROVED** [review]

File: component/button/button.css (hunk 1 above)
Comment (borkarsaish65) [reply component/button/button.css]: yes will do the changes

File: component/button/button.css (hunk 2 above)
Comment (borkarsaish65) [reply component/button/button.css:44]: yes understood. Thanks Ankur.

File: component/button/button.css (hunk 1 above)
Comment (borkarsaish65) [reply component/button/button.css]: Thanks

File: component/button/button.html (hunk 7 above)
Comment (borkarsaish65) [reply component/button/button.html]: ok I will correct the mistakes.

**Pull request merged**